init.lua

```diff
@@ -1,13 +1,22 @@
 require('vis')
 
 local function set_title(title)
+	-- print() cannot be used here as it will mess up vis
 	vis:command(string.format(":!printf '\\033]2;vis %s\\007'", title))
 end
 
+vis.events.subscribe(vis.events.INIT, function()
+	print('\27[22;2t')
```

Comment:
can you use the same value as in set_title for ESCAPE `\\033` ?

Reply:
no

Reply:
\033 is equvalent to \27 in lua, \033 cant be used because lua prefers the ***decimal*** representation.